datadog-crashtracker/data/libtest.c

```diff
@@ -0,0 +1,64 @@
+// Copyright 2025-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdio.h>
+
+int my_function(void) {
+    return 42;
+}
+
+// The following code helps in generating a case where blazesym failed at
+// opening the file and we were failing at normalizing/symbolizing the
+// address. 
+#define MAKE_FUNC(N) \
+    void func##N(void) { \
+        int arr[100]; \
+        for (int i = 0; i < 100; i++) { \
+            arr[i] = i * N; \
+        } \
+        printf("Function %d called, value = %d\n", N, arr[99]); \
+    }
+
+#define MAKE_STRUCT(N) \
+    struct Struct##N { \
+        int a[50]; \
+        double b[50]; \
+        char c[100]; \
+    };
+
+MAKE_STRUCT(1)
```

Comment:
comment explaining why we need so many of these

Reply:
explained in the comment above